packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGPanZoom/SVGPanZoom.js

```diff
@@ -117,22 +111,12 @@ function SVGPanZoom ({
     }
   }
 
-  function onWheel (event) {
-    const { deltaY } = event
-    if (deltaY < 0) {
-      onZoom('zoomout', -1)
-    } else {
-      onZoom('zoomin', 1)
-    }
-  }
-
   const { x, y, width, height } = scaledViewBox(zoom)
   const scale = imageScale(img)
 
   return (
     <div
```

Comment:
I think you can remove this `div` completely. I don’t think the ref is used anywhere else and the style could be applied to the SVG, if needed.

Reply:
My goal is to remove this entire component as noted in #2075. Any additional effort will be toward that.